pyproject.toml

```
@@ -1,12 +1,78 @@
 [build-system]
-requires = ["setuptools>=45", "wheel", "setuptools_scm>=6.2,<8"]
-build-backend = "setuptools.build_meta"
+requires = ["hatchling", "hatch-vcs"]
```

Since the license field is an SPDX expression, we're technically using [PEP 639](https://peps.python.org/pep-0639/) here.
```suggestion
requires = ["hatchling>=1.26.2", "hatch-vcs"]
```